sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClient.java

```diff
@@ -418,10 +418,11 @@ public final BlobInputStream openInputStream(BlobRange range, BlobRequestConditi
     }
 
     @Override
-    public BlobInputStream openInputStream(BlobInputStreamOptions options) {
+    public final BlobInputStream openInputStream(BlobInputStreamOptions options) {
         return openInputStream(options, null);
     }
 
+    @Override
     protected BlobInputStream openInputStream(BlobInputStreamOptions options, Context context) {
```

Comment:
This should be made `public`.

Reply:
Should the super method be public as well?